server/src/use-cases/version-control/azure-devops/code-review/code-review.js

```diff
@@ -50,29 +43,70 @@ export class CodeReview {
         authorId: pullRequest.createdBy.id,
         creationDate: pullRequest.creationDate,
         closedDate: pullRequest.closedDate,
-        reviewers: this.#parseReviewers(pullRequest.reviewers),
-        votesHistoryTimeline: this.#parseVotesTimeline(pullRequest.threads),
+        reviewers,
+        votesHistoryTimeline: this.#parseVotesTimeline(pullRequest.threads, reviewers, pullRequest.creationDate),
         threads: this.#parseThreads(pullRequest.threads),
         tags: this.#parseTags(pullRequest),
       };
     });
   }
 
-  static #parseReviewers(rawReviewers) {
+  static #parseReviewers(pullRequestThreads, rawReviewers, pullRequestCreationDate) {
     const reviewers = {};
 
+    const reviewersAddedTime = this.#parseReviewersAddedTime(pullRequestThreads);
+
     rawReviewers.forEach(({ id, displayName, vote, isRequired }) => {
       reviewers[id] = {
         author: displayName,
         vote: CODE_TO_VOTE.get(vote),
         isRequired: isRequired ?? false,
+        reviewerAddedTime: reviewersAddedTime[id] ?? pullRequestCreationDate,
       };
     });
 
     return reviewers;
   }
 
-  static #parseVotesTimeline(pullRequestThreads) {
+  static #parseReviewersAddedTime(pullRequestThreads) {
+    const policyAddedReviewersProperty = 'CodeReviewRequiredReviewerExampleReviewerIdentities';
+    const reviewerAddedProperty = 'CodeReviewReviewersUpdatedAddedIdentity';
```

Comment:
Is this the property value we get from azure response ?

Reply:
yes, Sathish.